manticore/core/state.py

```diff
@@ -234,3 +234,11 @@ def record_branches(self, targets):
                 self.branches[key] += 1
             except KeyError:
                 self.branches[key] = 1
+
+    def invoke_model(self, model):
+        '''
+        Invoke a model.
+
+        :param callable model: Model to invoke
+        '''
+        self.platform.invoke_model(model, (self,))  # TODO(mark): this can't support varargs core models!
```

Comment:
Explicitly specify `prefix_args` when passing `(self,)`? Also, move the comment on its own line?